src/Ocean/HydrostaticBoussinesq/OceanBoundaryConditions.jl

```diff
@@ -34,10 +33,39 @@ nothing needed to do since these are neumann BCs and no gradients here
 - `t`:  time, not used
 """
 @inline function ocean_boundary_state!(::HBModel, ::CoastlineFreeSlip,
-                                       ::Union{Rusanov,
-                                               CentralNumericalFluxGradient},
+                                       ::Rusanov,
                                        Q⁺, A⁺, n⁻, Q⁻, A⁻, t)
-  return nothing
+    u⁻ = Q⁻.u
+    n = @SVector [n⁻[1], n⁻[2]]
+    
+    Q⁺.u = u⁻ - 2 * (n∘u⁻) * n
```

Comment:
Do you really want to use `∘` here for dot product? `∘` is a julia function for function composition.

Reply:
Ah, I was using it instead of · (\cdot) because it's easier to read. Didn't realize it already had a syntactical meaning. I can switch back to using \circ.